### PR TITLE
github-issues-sheets-exporter

### PR DESCRIPTION
**Pull request by blaberry**
## Description
Add a issues exporter action

## Linked Issues
https://github.com/NeotokyoRebuild/paperwork/issues/13
- fixes # https://github.com/NeotokyoRebuild/paperwork/issues/13
- related #

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    github-project-issue-to-sheets:
+        runs-on: ubuntu-latest
+        name: github-project-issue-to-sheets
+        steps:
+        - name: github-project-issue-to-sheets
+          id: github-project-issue-to-sheets
+          uses: ViRGiL175/github-project-issue-to-sheets@dev
+          with:
+            google-api-service-account-credentials: ${{ secrets.GOOGLE_SERVICE_ACCOUNT_DATA }}
+            document-id: '1cuGeJlPIoKEo6su2vGx1wmkAi0l5o_7ZXCFH0dA5CAA'
```
Comment (Rainyan) [inline .github/workflows/main.yml:20]: Perhaps we could move `document-id` to https://github.com/NeotokyoRebuild/neo/settings/variables/actions so it doesn't have to be hardcoded inside the action?

**Review by Rainyan — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (blaberry) [reply .github/workflows/main.yml:20]: Good point, thanks

**Review by Rainyan — APPROVED** [review]

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  workflow_dispatch:
+  issues:
+    types: [opened, deleted, transferred, closed, reopened, assigned, unassigned, labeled, unlabeled]
+
+jobs:
+    github-project-issue-to-sheets:
+        runs-on: ubuntu-latest
+        name: github-project-issue-to-sheets
+        steps:
+        - name: github-project-issue-to-sheets
```
Comment (AdamTadeusz) [inline .github/workflows/main.yml:15]: suspicious hyphen here

**Review by AdamTadeusz — DISMISSED** [review]

File: .github/workflows/main.yml
Comment (Masterkatze) [inline .github/workflows/main.yml:1]: Rename `main.yml` so it reflect the content?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name: github-project-issue-to-sheets
+
+# Controls when the action will run. Triggers the workflow on push or pull request
```
Comment (Masterkatze) [inline .github/workflows/main.yml:3]: The comment is incorrect and not needed IMO.

File: .github/workflows/main.yml (hunk 2 above)
Comment (Rainyan) [reply .github/workflows/main.yml:15]: This is valid YAML/GH action syntax for a list as far as I can tell:

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idsteps
> A job contains a sequence of tasks called steps.

File: .github/workflows/main.yml
Comment (blaberry) [reply .github/workflows/main.yml:1]: done, ty.

File: .github/workflows/main.yml (hunk 3 above)
Comment (blaberry) [reply .github/workflows/main.yml]: smh this is why I shouldn't be allowed near a pastebuffer

**Review by Rainyan — APPROVED** [review]

**Review by Masterkatze — APPROVED** [review]

**Pull request merged**